typhos/display.py

```diff
@@ -1104,17 +1110,26 @@ def _move_display_to_layout(self, widget):
         self._scroll_area.setVisible(scrollable)
 
     def _generate_template_menu(self, base_menu):
-        """Generate the template switcher menu, adding it to ``base_menu``."""
+        """
+            Generate the template switcher menu, adding it to ``base_menu``.
+            Duplicate filenames will show full path.
+        """
```

Comment:
Multi-line docstring conventions have the indentation like this
```suggestion
        """
        Generate the template switcher menu, adding it to ``base_menu``.
        Duplicate filenames will show full path.
        """
```